lib/inspec/plugin/v2/installer.rb

```diff
@@ -340,15 +343,21 @@ def install_gem_to_plugins_dir(new_plugin_dependency, # rubocop: disable Metrics
         raise ex
       end
 
+      it_is_resource_pack_gem = new_plugin_dependency.name =~ /^(inspec)-/ && new_plugin_dependency.name =~ /-resources/
```

Comment:
Cosmetic change - Do you think this could be moved to a method like 

```
def is_resource_pack_gem?
  new_plugin_dependency.name =~ /^(inspec)-/ && new_plugin_dependency.name =~ /-resources/
end
```

and then invoke the method wherever `it_is_resource_pack_gem` variable is used.